presto-mongodb/src/test/java/io/prestosql/plugin/mongodb/BaseMongoDistributedQueries.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Mention supported versions to docs.

Reply:
Put them in a separate commit, let me know if I should roll them in to the second commit.